### ISSUE TITLE
Keeps wrong indentation

### ISSUE THREAD

**Issue by remy727** [rubocop/rubocop#11500]
## Expected behavior

Describe here how you expected RuboCop to behave in this particular situation.

```ruby
class BillingsController < ApplicationController
  def update
    render(
      json: {
        success: false,
        errors: [{
          name: 'billing',
          messages: ['Billing record does not belong to organization']
        }]
      },
      status: :bad_request
    )
  end
end

```

## Actual behavior

```ruby
class BillingsController < ApplicationController
  def update
    render(
      json:
                    {
                      success: false,
                      errors: [{
                        name: 'billing',
                        messages: ['Billing record does not belong to organization']
                      }]
                    },
      status: :bad_request
    )
  end
end
```

## Steps to reproduce the problem

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
$ [bundle exec] rubocop -V
1.44.0 (using Parser 3.2.0.0, rubocop-ast 1.24.1, running on ruby 3.2.0) [x86_64-darwin22]
```

**Comment by koic** [comment]
I can't reproduce it. Can you provide repro steps and your .rubocop.yml custom config?

**Comment by remy727** [comment]
@koic, sure.

You should reproduce without `.rubocop.yml`

### Reproduct steps
1. Install/use ruby `3.2.0`.
2. Install rubocop.
```bash
$ gem install rubocop
Fetching rubocop-1.44.1.gem
Successfully installed rubocop-1.44.1
Parsing documentation for rubocop-1.44.1
Installing ri documentation for rubocop-1.44.1
Done installing documentation for rubocop after 12 seconds
1 gem installed
```
3. Create `test.rb` with the below code:
```ruby
# frozen_string_literal: true

# Billings Controller
class BillingsController < ApplicationController
  def update
    render(
      json:
                    {
                      success: false,
                      errors: [{
                        name: 'billing',
                        messages: ['Billing record does not belong to organization']
                      }]
                    },
      status: :bad_request
    )
  end
end
```
4. Lint
```bash
$ rubocop -A test.rb

Inspecting 1 file
C

Offenses:

test.rb:5:3: C: Metrics/MethodLength: Method has too many lines. [11/10]
  def update ...
  ^^^^^^^^^^

1 file inspected, 1 offense detected
```
